arcgis-ios-sdk-samples/Scenes/Animate 3D graphic/Animate3DGraphicViewController.swift

```diff
@@ -388,16 +383,20 @@ extension Animate3DGraphicViewController: UIAdaptivePresentationControllerDelega
     }
 }
 
-private class Frame {
+struct Frame {
```

Comment:
Making this a struct eliminates the overhead of memory management for these object, which is a nice and free improvement. The type could probably do with a better name, but I had to stop experimenting somewhere.